docs/user/candidates.rst

```diff
@@ -19,11 +19,55 @@ Core Objects
 
 These are Fonduer_'s core objects used for Mention and Candidate extraction.
 
-.. automodule:: fonduer.candidates
+.. autoclass:: fonduer.candidates.MentionExtractor
     :members:
     :inherited-members:
     :show-inheritance:
 
+.. autoclass:: fonduer.candidates.CandidateExtractor
+    :members:
+    :inherited-members:
+    :show-inheritance:
+
+MentionSpaces
+-------------
+
+A *MentionSpace* defines a space where mentions lie on. You can use a pre-defined child
```

Comment:
> A *MentionSpace* defines a space where mentions lie on.

I wonder if we can clarify this statement. Maybe something like, "defines the space of mentions considered by Fonduer"? Any thoughts here?

Reply:
I think the first statement should tell the concept of MentionSpace without talking about any particular implementation. Any following ones can talk about Fonduer's implementation.
How about this?
"A *MentionSpace* defines the space of mentions, i.e., the set of all possible mentions."

Reply:
That sounds great! If you update that I think this PR is ready to merge.

Reply:
Thanks! I modified the 1st statement, (and the 2nd a little bit).